src/main/asciidoc/_chapters/cp.adoc

```diff
@@ -193,6 +209,12 @@ link:https://issues.apache.org/jira/browse/HBASE-5448[HBASE-5448]). To upgrade y
 HBase cluster from 0.94 or earlier to 0.96 or later, you need to reimplement your
 coprocessor.
 
+In HBase 2.0.0, we make use of a shaded version of protobuf 3.x, but still keep the
+protobuf for coprocessor on 2.5.0. In 3.0.0, we removed all dependencies on non-shaded
```

Comment:
s/In HBase 2.0.0/In HBase.2.x/

s/make/made/

s/still keep/kept/

s/coprocessor/coprocessors/